Velaptor/Services/IDotnetService.cs

```diff
@@ -0,0 +1,27 @@
+namespace Velaptor.Services;
+
+/// <summary>
+/// Invokes Dotnet functions.
+/// </summary>
+public interface IDotnetService
+{
+    /// <summary>
+    /// Invokes System.GC.KeepAlive method.
+    /// </summary>
+    /// <param name="obj">The object to reference.</param>
```

Comment:
Please refactor the code doc to reference the original functionality that it wraps.

```suggestion
    /// <inheritdoc cref="GC.KeepAlive"/>
```